lib/Data/Chronicle/Mock.pm

```diff
@@ -49,7 +49,8 @@ sub get_mocked_chronicle {
 
     my $chronicle_w = Data::Chronicle::Writer->new(
         cache_writer    => $redis,
-        db_handle       => $dbh);
+        db_handle       => $dbh,
+        ttl             => 86400);
```

Comment:
As I have mentioned 1 day ttl is dangerous for us. We will be wiping out our data over the weekend and Monday redis will be empty. I would suggest 1 week as a safe bet.